src/protocol/libp2p/identify.rs

```diff
@@ -183,8 +183,14 @@ pub(crate) struct Identify {
     /// User agent.
     user_agent: String,
 
+    /// Listen addresses.
+    listen_addresses: ListenAddresses,
+
     /// Public addresses.
-    listen_addresses: PublicAddresses,
+    public_addresses: PublicAddresses,
+
+    /// User provided list of addresses.
+    user_addresses: Vec<Vec<u8>>,
```

Comment:
I don't think we need one more place to specify public addresses. Getting public addresses via a handle from `TransportManager` should be enough.

Reply:
Oki doki, have removed these thanks! 🙏

Reply:
I probably was not quite clear on this, but I meant we don't need to let user pass addresses directly to Identify config if we already have an API to notify litep2p about public addresses.